cime_config/buildnml

```diff
@@ -237,6 +237,11 @@ endif
 set DO_OALK = .false.
 set OALKSCEN = "''"
 set OALKFILE = "''"
+set ADDALK    = 0.135
+set CDRMIP_LATMAX = 70.0
+set CDRMIP_LATMIN = -60.0
+set RAMP_START    = 2025
+set RAMP_END      = 2035
```

Comment:
We haven't done it before, but I wonder if it would make sense to introduce a naming convention for these variables on the form: `NAMELIST`_`VARIABLE`
to make it clear where the variable belongs. Not sure if this is the best way, but I think it can be useful.